sequencer/.env

```diff
@@ -0,0 +1,5 @@
+BootCoordinator=
+BootCoordinatorURL=
+PGUSER=postgres
+PGPASSWORD=123
+PGDATABASE=statedb
```

Comment:
I think we should remove `.env` from prod. I added it to `.gitignore` and created `.env.example`. Also mentioned to `cp .env.example .env` in the sequencer readme

Reply:
Critical comments.
I have removed .env file.